networks/suzuka/suzuka-config/src/execution_extension.rs

```diff
@@ -0,0 +1,29 @@
+use godfig::env_default;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Config {
+	#[serde(default = "default_block_retry_count")]
+	pub block_retry_count: u64,
+
+	#[serde(default = "default_block_retry_increment_microseconds")]
+	pub block_retry_increment_microseconds: u64,
+}
+
+impl Default for Config {
+	fn default() -> Self {
+		Self {
+			block_retry_count: default_block_retry_count(),
+			block_retry_increment_microseconds: default_block_retry_increment_microseconds(),
+		}
+	}
+}
+
+env_default!(default_block_retry_count, "BLOCK_RETRY_COUNT", u64, 10);
```

Comment:
Should there be a distinctive prefix for env variables like `MOVEMENT_BLOCK_RETRY_COUNT`?

Reply:
Yeah, this should probably be prefixed with `SUZUKA_`.